modules/api/api_gateway.tf

```diff
@@ -1,9 +1,3 @@
-resource "aws_api_gateway_resource" "graphql_endpoint" {
```

Comment:
Have all the resources that support tags got them added?

Reply:
Yes. `aws_api_gateway_stage` is the only API Gateway resource is the only one that accepts tags, and it's got them.

I'm a bit surprised that `api_gateway_rest_api` doesn't have tags in Terraform, because it does have tags in the UI.